pkg/client/client_test.go

```diff
@@ -747,3 +661,10 @@ func getRandomPort() (int, error) {
 
 	return listener.Addr().(*net.TCPAddr).Port, nil
 }
+
+// Extract and return vaultLocation from vaultLocationURL: /encrypted-data-vaults/{vaultLocation}
```

Comment:
The vault location is the entire URL - what you're actually doing is returning the vault ID/name.